clients/stellar-relay-lib/src/connection/overlay_connection.rs

```diff
@@ -22,6 +22,15 @@ pub struct StellarOverlayConnection {
 	is_disconnected: bool,
 }
 
+impl StellarOverlayConnection {
+	pub fn get_actions_sender(&self) -> mpsc::Sender<ConnectorActions> {
+		self.actions_sender.clone()
+	}
+	pub fn disconnect_status(&self) -> ConnectorActions {
```

Comment:
I'd rename this to `get_disconnect_action()` to make it clearer that the function returns the `ConnectorAction` to disconnect.

Reply:
You are right. done.